src/bin/tikv-server.rs

```diff
@@ -129,10 +129,8 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
     }
 
     // Initialize raftstore channels.
-    let mut event_loop = store::create_event_loop(&cfg.raft_store)
-        .unwrap_or_else(|e| fatal!("failed to create event loop: {:?}", e));
-    let store_sendch = SendCh::new(event_loop.channel(), "raftstore");
-    let (significant_msg_sender, significant_msg_receiver) = mpsc::channel();
+    let (router, system) = fsm::create_raft_batch_system(&cfg.raft_store);
```

Comment:
Seems you forget to spawn this system?

Reply:
It's spawn inside `node.rs` instead of here.